src/components/zen-menu/zen-menu.scss

```
@@ -0,0 +1,13 @@
+@import '../scss/component-helpers.scss';
+
+:host {
+  display: block;
+  position: absolute;
```

Set default width as per design here:
```suggestion
  position: absolute;
  width: 14.125rem;
```

---

done